Islandora-CLAW/islandora
# Don't translate mimetypes

**GitHub Issue**: Resolves Islandora-CLAW/CLAW#840

# What does this Pull Request do?

By setting up content translation on your site, fields indicated as non-translatable will no longer get language tags applied to them by `jsonld`.  Big @DiegoPino++ on this one.  No code changes neccessary, just config!

Now we can pull our derivatives out of Fedora again!

# What's new?
A bunch of exported config into our features
* Content translation settings
* Field settings
* Some field overrides

# How should this be tested?

- Make a Media
- Check out its jsonld, the mimetpe property should have a lang of '@en'
```json
{

         "http:\/\/www.ebu.ch\/metadata\/ontologies\/ebucore\/ebucore#hasMimeType":[
            {
               "@value":"image\/gif",
               "@language":"en"
            }
         ],
```
- Enable the core `content_translation` module
- Pull in these changes
- Import both features
- Go check out the jsonld again, this time mimetype should be explictly labeled as an `xsd:string` and should not have any languages applied to it
```json
         "http:\/\/www.ebu.ch\/metadata\/ontologies\/ebucore\/ebucore#hasMimeType":[
            {
               "@value":"image\/gif",
               "@type":"http:\/\/www.w3.org\/2001\/XMLSchema#string"
            }
         ],
```

# Interested parties
Here's an easy one @Islandora-CLAW/committers
